src/interfaces/package-manager.interfaces.ts

```diff
@@ -26,6 +32,10 @@ export interface PackageDependencyTransport {
     name: string;
     version: string;
     rootNodeId: string;
+    external: boolean
+    draftId: string;
+    updateAvailable: boolean;
```

Comment:
Can we include this field in the listing as well? @frkntrcnn this can be used directly to warn users of dependencies not using latest active version.

Reply:
@promeris agree, could be good to show it with just using this, instead of comparing versions